SU-HSDO/suhumsci
# HSD8-500 Created UI for events importer to change URLs

# READY FOR REVIEW

# Summary
- UI for events importer
- removed site specific configs

# Needed By (Date)
- soon

# Urgency
- medium

# Steps to Test

1. Checkout this branch
1. Clear caches
1. Go do the events importer form at /admin/structure/migrate/events
1. Add an event url
1. Run importer
1. Verify new events content

- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
